addons/mass_mailing/static/src/js/mass_mailing_snippets.js

```diff
@@ -147,6 +130,14 @@ options.registry.ImageTools.include({
         }
         return this._super(...arguments);
     },
+
+    _computeWidgetVisibility(widgetName, params) {
+        // Transform is _very_ badly supported in mail clients. Hide the option.
+        if (widgetName === 'image_transform_opt') {
+            return this.snippets && this.snippets.split('.')[0] !== "mass_mailing";
+        }
+        return this._super(...arguments);
```

Comment:
Whoa... why would this file affect the website editor? Is not the mass mailing editor still inside an iframe? Why is this fine loaded at all for the website edition?